src/components/Footer/Footer.scss

```diff
@@ -0,0 +1,73 @@
+@use '../../utils/variables' as *;
+@use '../../utils/mixins' as *;
+
+.footer {
+  @include grid;
+
+  row-gap: 32px;
+
+  margin-top: 64px;
+  padding: 32px 16px;
+  border-top: 1px solid $color-elements;
+
+  &__logo {
+    grid-column: 1 / 5;
+    height: 33px;
+  }
+  &__nav {
+    grid-column: 1 / 5;
+  }
+  &__top-link {
+    grid-column: 1 / 5;
+  }
+}
+
+.footer-nav {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 16px;
```

Comment:
you can use variables for your gaps, it`s good for calculate in future your container width